test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -55,18 +55,19 @@ def test_successful_purchase_with_more_options
     assert_equal 'Succeeded', response.message
   end
 
-  # When approval is required, purchase will not make a capture request
+  # When requires_approval is true (or not present),
+  # `purchase` will make both an `auth` and # a `capture` call
```

Comment:
Stray octothorpe:
```suggestion
  # `purchase` will make both an `auth` and a `capture` call
```

Reply:
🦅 👁 I appreciate the thorough reviewa lot! Thanks, that pound sign is outta here.